backend/schedule/models.py

```diff
@@ -131,6 +133,21 @@ class ScheduleItem(TimeStampedModel):
         on_delete=models.PROTECT,
     )
 
+    maximum_capacity = models.PositiveIntegerField(
+        _("maximum capacity"), blank=True, null=True
+    )
+
+    @property
+    def allows_booking(self):
+        return self.type == ScheduleItem.TYPES.training
```

Comment:
we need to take into account also SUBMISSION types where the submission is a training :)